lutris/gui/lutriswindow.py

```diff
@@ -688,7 +692,14 @@ def on_add_game_button_clicked(self, *args):
             callback=lambda: self.add_game_to_view(dialog.game.id)
         )
         return True
-
+    
+    @GtkTemplate.Callback
```

Comment:
what the fuck.
why tis there an action for that

Reply:
oups, tbh was just copying add_game and i didn't notice, I'll just remove it

Reply:
removed